python/lsst/meas/algorithms/matcherSourceSelector.py

```diff
@@ -26,7 +26,9 @@
 
 from lsst.afw import table
 import lsst.pex.config as pexConfig
-from .sourceSelector import BaseSourceSelectorConfig, BaseSourceSelectorTask, sourceSelectorRegistry
+from .sourceSelector import BaseSourceSelectorConfig
+from .sourceSelector import BaseSourceSelectorTask
+from .sourceSelector import sourceSelectorRegistry
```

Comment:
This seems far more verbose than the original, which can use a line continuation character if need be